packages/debugger/src/panels/callstack/body.tsx

```diff
@@ -63,15 +63,34 @@ const FramesComponent = ({
     };
   }, [model]);
 
+  const toShortLocation = (el: IDebugger.IStackFrame) => {
+    let path = el.source?.path || '';
+    const lastIndex = path.lastIndexOf('/');
+    const secondLastIndex = path.lastIndexOf('/', lastIndex - 1);
+    // If the we end up finding the leading '/', leave it in
+    path = path.substring(secondLastIndex > 0 ? secondLastIndex + 1 : 0);
```

Comment:
As a short location, do we want to have "last dirname + file name with extension"?

Wondering whether it would be possible to use a combination of `PathExt.basename` and `PathExt.dirname` instead of manually doing a substring on the raw path?

For example (although not tested) the containing directory could be found with something like `PathExt.basename(PathExt.dirname(path))`. And the file name with `PathExt.basename(path)`.

Reply:
Yeah, that's much better! Used your suggestion.

FWIW, while chrome does not does this, I found it unusable in python without last dirname. Some of this is due to how xpython names cells and some of it is the amount of code people put in `_init_.py` and similar.